source/particle/property/velocity_gradient.cc

```diff
@@ -0,0 +1,98 @@
+/*
+  Copyright (C) 2015 - 2025 by the authors of the ASPECT code.
+ This file is part of ASPECT.
+ ASPECT is free software; you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation; either version 2, or (at your option)
+ any later version.
+ ASPECT is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+ You should have received a copy of the GNU General Public License
+ along with ASPECT; see the file LICENSE.  If not see
+ <http://www.gnu.org/licenses/>.
+ */
+
+#include <aspect/particle/property/velocity_gradient.h>
+#include <aspect/particle/property/interface.h>
+#include <aspect/simulator_access.h>
```

Comment:
no need for these two includes here, they are already included in the header file.